src/textarea/examples.js

```diff
@@ -90,9 +90,11 @@ export default {
       <React.Fragment>
         <Textarea size={SIZE.compact} placeholder="Placeholder" />
         <br />
+        {/* eslint-disable-next-line jsx-a11y/no-autofocus */}
+        <Textarea size={SIZE.compact} placeholder="Focused" autoFocus />
+        <br />
         <Textarea
           initialState={{value: 'uber'}}
-          autoFocus
```

Comment:
`autoFocus` is a part of functionality form controls provide, we should be able to keep these examples